src/library-authoring/components/ComponentCard.tsx

```diff
@@ -21,39 +23,47 @@ type ComponentCardProps = {
   blockTypeDisplayName: string,
 };
 
-const ComponentCardMenu = () => (
-  <Dropdown>
-    <Dropdown.Toggle
-      as={IconButton}
-      src={MoreVert}
-      iconAs={Icon}
-      variant="primary"
-    />
-    <Dropdown.Menu>
-      <Dropdown.Item disabled>
-        <FormattedMessage
-          {...messages.menuEdit}
-        />
-      </Dropdown.Item>
-      <Dropdown.Item disabled>
-        <FormattedMessage
-          {...messages.menuCopyToClipboard}
-        />
-      </Dropdown.Item>
-      <Dropdown.Item disabled>
-        <FormattedMessage
-          {...messages.menuAddToCollection}
-        />
-      </Dropdown.Item>
-    </Dropdown.Menu>
-  </Dropdown>
-);
+const ComponentCardMenu = ({ usageKey }: { usageKey: string }) => {
+  const intl = useIntl();
+  const { showToast } = useContext(ToastContext);
+  const updateClipboardClick = () => {
+    updateClipboard(usageKey)
+      .then(() => showToast(intl.formatMessage(messages.copyToClipboardSuccess)))
+      .catch(() => showToast(intl.formatMessage(messages.copyToClipboardError)));
+  };
+
+  return (
+    <Dropdown id="component-card-dropdown">
+      <Dropdown.Toggle
+        id="component-card-menu-toggle"
+        as={IconButton}
+        src={MoreVert}
+        iconAs={Icon}
+        variant="primary"
+        alt="component-card-menu-toggle" // FixMe: Add alt text
+        data-testid="component-card-menu-toggle"
+      />
+      <Dropdown.Menu>
+        <Dropdown.Item disabled>
+          {intl.formatMessage(messages.menuEdit)}
+        </Dropdown.Item>
+        <Dropdown.Item onClick={updateClipboardClick} data-testid="component-card-menu-copy-clipboard">
```

Comment:
nit: Is it possible to get the dropdown item in the tests through other means instead of relying on the data-testid? Since that is generally discouraged and should be the last resort.

Reply:
Unfortunately not. :disappointed: 
The card menu is a button without a label. Getting via `getByRole('button', { name: '' })` could work, but I don't think it will have a better result in the test readability.

For the Dropdown.Item, I'm using the label. Removing the data-testid here as it is not being used: b960fc53d52d70aaa2c91e40c6a4d6add20db1e9

Reply:
Got it, thanks for the clarification